packages/ckeditor5-dev-utils/lib/index.js

```diff
@@ -10,5 +10,8 @@ module.exports = {
 	logger: require( './logger' ),
 	tools: require( './tools' ),
 	stream: require( './stream' ),
-	workspace: require( './workspace' )
+	workspace: require( './workspace' ),
+	getAllTranslations: require( './translations/getalltranslations' ),
+	createTranslator: require( './translations/createtranslator' ),
```

Comment:
What's `createTranslator()`? We need a function which for a given source code returns a new source code with all `t()` calls replaced. We may later think on how to integrate source map synchronisation with that – perhaps this function should accept a callback which is executed for each found `t()` or something, but it must provide you all data already. 

Besides, wrap the translation tools in an object, just like the other tools are wrapped.